docker-compose.yml

```diff
@@ -2,18 +2,21 @@ version: '3'
 
 services:
   chatserver:
-    build:
-      context: ./server
-      dockerfile: Dockerfile
+    build: ./server/
+    volumes:
+      - ./server:/usr/app
     ports:
       - 9081:9081
 
   chatclient:
-    build:
-      context: ./client
-      dockerfile: Dockerfile
+    build: ./client/
+    volumes:
+      - ./client:/usr/app
     ports:
       - 8080:8080
 
   rabbitmq:
-    image: rabbitmq:3.7.4
+    image: rabbitmq:3-management
+    ports:
+      - 5672:5672
+      - 15672:15672
```

Comment:
added the management so we can have access to the management plugin, that will give access to rabbit in the browser when accessing localhost:15762
- ports stands for: map my 15672 to yours : 15762. If we don't specify a port, well, you can't access it. If you only specify 
ports: 
- 15672 , that will assign a random port for us. so we have to  run docker container ps, then it will be listed the port there.
On that note, there is the EXPOSE that will you see in the Dockerfile, that's just good practice that's letting people know which ports are open to be used, but that doesn't actually open anything. So EXPOSE without the -ports is useless.